### PR TITLE
Changes the default ghost lighting, makes it a preference

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request

I think the way ghost lighting looks right now is really crummy.
It's dark enough you can see where the shadows should be, but it's just bright enough for everything to look like dog poo

A lot of what makes the game look nice is the depth of the lighting and if we just hide that for observers we're shooting ourselves in the foot.

I'm also making it a game preference, so if someone wants to have bad opinions they can easily.

## Why It's Good For The Game

https://user-images.githubusercontent.com/58055496/157390981-40758fbc-480e-43be-8a91-0234d68f423f.mp4

## Changelog

:cl:
add: Ghost darkness is now a game preference, and will persist between rounds
balance: Ghost darkness defaults to a bit darker then currently. I think it looks better
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
I wish I had tweak:

**Comment by Colovorat** [comment]
> I wish I had tweak:

You don't

File: code/modules/client/preferences/ghost_lighting.dm [hunk 1]
```diff
@@ -0,0 +1,28 @@
+GLOBAL_LIST_INIT(ghost_lighting_options, list(
+		"Fullbright" = LIGHTING_PLANE_ALPHA_INVISIBLE,
```
Comment (Mothblocks) [inline code/modules/client/preferences/ghost_lighting.dm:2]: Indent once

File: code/modules/client/preferences/ghost_lighting.dm [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+		"Night Vision" = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE,
+		"Darker" = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE,
+		"Normal" = LIGHTING_PLANE_ALPHA_VISIBLE
+	))
```
Comment (Mothblocks) [inline code/modules/client/preferences/ghost_lighting.dm:6]: Remove indent

File: code/modules/client/preferences/ghost_lighting.dm [hunk 3]
```diff
@@ -0,0 +1,28 @@
+GLOBAL_LIST_INIT(ghost_lighting_options, list(
+		"Fullbright" = LIGHTING_PLANE_ALPHA_INVISIBLE,
+		"Night Vision" = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE,
+		"Darker" = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE,
+		"Normal" = LIGHTING_PLANE_ALPHA_VISIBLE
```
Comment (Mothblocks) [inline code/modules/client/preferences/ghost_lighting.dm:5]: Trailing comma

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Comment by LemonInTheDark** [comment]
@Mothblocks I think this is right? I'm honestly not sure

**Review by Mothblocks — APPROVED** [review]
Yeah, it's just making sure you don't indent more than once

File: code/modules/mob/mob.dm [hunk 4]
```diff
@@ -220,7 +220,7 @@
 			if(M != loc) // Only give the blind message to hearers that aren't the location
 				msg = blind_message
 				msg_type = MSG_AUDIBLE
-		else if(M.lighting_alpha > LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE && T.is_softly_lit() && !in_range(T,M)) //if it is too dark, unless we're right next to them.
+		else if(M.lighting_alpha > LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE && T.is_softly_lit() && !in_range(T,M) && !isobserver(M)) //if it is too dark, unless we're right next to them.
```
Comment (AnturK) [inline code/modules/mob/mob.dm:223]: This should probably be some trait/flag so it's more obvious why this check is here. Or add a comment at least.

**Review by AnturK — CHANGES_REQUESTED** [review]

File: code/modules/mob/mob.dm (hunk 4 above)
Comment (LemonInTheDark) [reply code/modules/mob/mob.dm:223]: Good point

**Pull request merged**